app/components/e-panel.vue

```diff
@@ -0,0 +1,543 @@
+    &__content--spacing-top-100 {
+      padding-top: $spacing--10;
+    }
+
+    &__headline--spacing-top-200,
+    &__content--spacing-top-200 {
+      padding-top: $spacing--15;
+    }
+
+    &__headline--spacing-top-300,
+    &__content--spacing-top-300 {
+      padding-top: $spacing--20;
+    }
+
+    &__headline--spacing-top-400,
+    &__content--spacing-top-400 {
+      padding-top: $spacing--30;
+    }
+
+    &__headline--spacing-right-000,
+    &__content--spacing-right-000 {
+      padding-right: $spacing--0;
+    }
+
+    &__headline--spacing-right-100,
+    &__content--spacing-right-100 {
+      padding-right: $spacing--10;
+    }
+
+    &__headline--spacing-right-200,
+    &__content--spacing-right-200 {
+      padding-right: $spacing--15;
+    }
+
+    &__headline--spacing-right-300,
+    &__content--spacing-right-300 {
+      padding-right: $spacing--20;
+    }
+
+    &__headline--spacing-right-400,
+    &__content--spacing-right-400 {
+      padding-right: $spacing--30;
+    }
+
+    &__headline--spacing-bottom-000,
+    &__content--spacing-bottom-000 {
+      padding-bottom: $spacing--0;
+    }
+
+    &__headline--spacing-bottom-100,
+    &__content--spacing-bottom-100 {
+      padding-bottom: $spacing--10;
+    }
+
+    &__headline--spacing-bottom-200,
+    &__content--spacing-bottom-200 {
+      padding-bottom: $spacing--15;
+    }
+
+    &__headline--spacing-bottom-300,
+    &__content--spacing-bottom-300 {
+      padding-bottom: $spacing--20;
+    }
+
+    &__headline--spacing-bottom-400,
+    &__content--spacing-bottom-400 {
+      padding-bottom: $spacing--30;
+    }
+
+    &__headline--spacing-left-000,
+    &__content--spacing-left-000 {
+      padding-left: $spacing--0;
+    }
+
+    &__headline--spacing-left-100,
+    &__content--spacing-left-100 {
+      padding-left: $spacing--10;
+    }
+
+    &__headline--spacing-left-200,
+    &__content--spacing-left-200 {
+      padding-left: $spacing--15;
+    }
+
+    &__headline--spacing-left-300,
+    &__content--spacing-left-300 {
+      padding-left: $spacing--20;
+    }
+
+    &__headline--spacing-left-400,
+    &__content--spacing-left-400 {
+      padding-left: $spacing--30;
+    }
+
+    &__triangle-button {
+      position: absolute;
+      bottom: 0;
+      right: 0;
+      border: none;
+      cursor: pointer;
+    }
+
+    &__triangle-button--size-small {
+      width: 22px;
+      height: 22px;
+    }
+
+    &__triangle-button--size-big {
+      width: 33px;
+      height: 33px;
+    }
+
+    &__triangle-button--contour-thin {
+      bottom: -1px;
+    }
+
+    &__triangle-button--contour-thick {
+      bottom: -2px;
+    }
+
+    &__triangle-button--color-yellow {
+      background-color: $color-primary--1;
+    }
+
+    &__triangle-button--color-blue {
+      background-color: $color-secondary--1;
+    }
+
+    &__triangle-button::after {
+      content: "";
+      display: block;
+      position: absolute;
+      box-shadow: 0 2px 2px -2px rgba(0, 0, 0, 0.3);
```

Comment:
- [ ] Please also add a `transition` for the shadow changes.

Reply:
- [ ] You can also use `$color` variables inside `rgba()`

Reply:
Let's discuss on tuesday.